WikiLookup.json

```diff
@@ -575,5 +575,46 @@
 			"Star Citizen",
 			"Squadron 42"
 		]
+	},
+		{
+		"name": "Unofficial Elder Scrolls Pages",
+		"homepage": "https://en.uesp.net	",
+		"lang": "en",
+		"api": "https://en.uesp.net/w/api.php",
+		"companies": [
+		    "Bethesda Softwarks",
```

Comment:
As this wiki only covers a subset of the games Bethesda/Zenimax have created, it shouldn't include a `companies` entry.